loader/loader.go

```diff
@@ -1272,8 +1420,18 @@ func (l *Loader) restoreData(ctx context.Context) error {
 				dispatchMap[fmt.Sprintf("%s_%s_%s", db, table, file)] = j
 			}
 		}
+
+		for view := range views {
+			viewFile := fmt.Sprintf("%s/%s.%s-schema-view.sql", l.cfg.Dir, db, view)
+			l.logger.Info("start to create view", zap.String("view file", viewFile))
+			err := l.restoreView(ctx, dbConn, viewFile, db, view)
+			if err != nil {
+				return err
+			}
+			l.logger.Info("finish to create view", zap.String("view file", viewFile))
+		}
```

Comment:
What if some views needed tables in other databases are not created yet? I think we should create views when all tables in all databases have been set up correctly.

Reply:
done in https://github.com/pingcap/dm/pull/1310/commits/d16871614a6202d7f5414827a86276e1bf1dbcba